pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -429,7 +429,11 @@ public void unloadNamespaceBundlesGracefully() {
             // make broker-node unavailable from the cluster
             if (pulsar.getLoadManager() != null) {
                 try {
-                    pulsar.getLoadManager().get().disableBroker();
+                    if (pulsar.getLoadManager() == null || pulsar.getLoadManager().get() == null) {
```

Comment:
can you move the condition to line 430? since line 430 already checks `pulsar.getLoadManager() != null`.

Reply:
No problem with that. It simplifies things.